### ISSUE TITLE
Accessibility: MAS42A: High Contrast: The image is missing for "Mac" download link and "Search Docs" edit box in High contrast white Theme

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#14]
#A11yMAS
#MAS42A
#Wipro
#vscode-website
#Firefox

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Prerequisite:**
**_1. Change system setting to High contrast White_**
    Go to Settings-->Ease of Access-->High Contrast.
    Select "High Contrast White" from the "Choose a theme" dropdown. Click on Apply.

**Repro Steps:**
**_Scenario-1:_**
1. Open [https://code.visualstudio.com/](https://code.visualstudio.com/) in edge.
2. Navigate to the "Mac" package download link below the "Get started now" above the footer of the page.
3. Observe the image above the Mac package download link in high contrast white theme.
Note: Same issue is observed for the below link:
[http://code.visualstudio.com/Download](http://code.visualstudio.com/Download)

**_Scenario-2:_**
1. Open [https://code.visualstudio.com/](https://code.visualstudio.com/) in edge.
2. Naviagte to the "Search docs"  box beside the "Download" link in header of the page .
3. Observe the Search image is missing in high contarst white.

Note: Same issue observed in below links:
[http://code.visualstudio.com/docs
](http://code.visualstudio.com/docs
)http://code.visualstudio.com/insiders
https://code.visualstudio.com/updates
https://code.visualstudio.com/docs/supporting/faq
https://code.visualstudio.com/docs/introvideos/overview
https://code.visualstudio.com/docs/introvideos/overview
https://code.visualstudio.com/docs/editor/accessibility
https://code.visualstudio.com/docs/editor/codebasics
https://code.visualstudio.com/docs/customization/userandworkspace
https://code.visualstudio.com/docs/languages/overview
https://code.visualstudio.com/docs/setup/setup-overview
https://code.visualstudio.com/docs/runtimes/nodejs

**Actual Result:**
**_Scenario-1:_**
The image is not visible for the Mac download package link in high contrast white.

**_Scenario-2:_**
The search image is missing in the "Search docs" box beside the download link in high contrast white.

**Expected Result:**
**_Scenario-1:_**
The image should be visible above the Mac package download link in high contrast white.

**_Scenario-2:_**
There should be a search image in the "Search docs" box beside download link in high contrast white as visible in normal mode

**User Impact:**
The persons with colour blindness and low vision will not be able to know that there is a image for the link/ Serach box

**Screenshots:**
[Image not visible in HCW.zip](https://github.com/wiproaccess/vscode-website/files/687070/Image.not.visible.in.HCW.zip)
![in black backgroung images are disabled for smoke pass](https://cloud.githubusercontent.com/assets/24825441/21678551/4d3db94a-d365-11e6-8d28-f0d630ad5513.png)
[Sc 2 Search image is missing in HCW.zip](https://github.com/wiproaccess/vscode-website/files/687152/Sc.2.Search.image.is.missing.in.HCW.zip)

**Comment by waderyan** [comment]
Fixed with de3202358747b808a106c02b2f20f84eb819670d

**Comment by waderyan** [comment]
Another fix with 2b4e904688206594e2c4cbcb33974566b74459ff

**Comment by wiproaccess** [comment]
The issue is verified hence adding the label 'closed' and closing the bug.
